lib/stripe/api_operations/request.rb

```diff
@@ -3,8 +3,19 @@ module APIOperations
     module Request
       module ClassMethods
         OPTS_KEYS_TO_PERSIST = Set[:api_key, :api_base, :client, :stripe_account, :stripe_version]
```

Comment:
I think this line gives us another hint for an `opt` that we can warn on! What do you think about adding `stripe_version` to the set?